### DISCUSSION TITLE
Feature REQ: Hotkey zum Ein-/Ausblenden der Bildschirmtastatur

### DISCUSSION THREAD

**Discussion by 555wolfman** [Rojetto/ReNeo#71] [Ideas]
Gerade ganz am Anfang wenn man Neo/Bone übt jedoch doch öfters auf QWERTZ hin und her schalten muss benötigt man die Bildschirmtastatur oder dann alternativ den Platz am Bildschirm.

Daher wäre es sehr hilfreich, wenn man die **Bildschirmtastatur** einfach per **Hotkey** **aktivieren/deaktivieren** könnte.

Danke

lg
Olaf

**Comment by Rojetto** [comment]
Den gibt es bereits, probier mal `Mod3+F1`. Falls das nicht passt, kannst du in der `config.json` auch ein anderes Tastenkürzel definieren, siehe dazu https://github.com/Rojetto/ReNeo#allgemeine-konfiguration

**Reply by 555wolfman** [reply]
Danke!
